themes/src/main/resources/theme/base/login/register.ftl

```diff
@@ -150,9 +145,20 @@
                     </div>
                 </div>
 
-                <div id="kc-form-buttons" class="${properties.kcFormButtonsClass!}">
-                    <input class="${properties.kcButtonClass!} ${properties.kcButtonPrimaryClass!} ${properties.kcButtonBlockClass!} ${properties.kcButtonLargeClass!}" type="submit" value="${msg("doRegister")}"/>
-                </div>
+                <#if recaptchaRequired?? && !(recaptchaVisible!false)>
+                    <script type="text/javascript">function onSubmitRecaptcha(token) { document.getElementById("kc-register-form").submit(); }</script>
```

Comment:
For new scripts, we default to the [`module`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#applying_the_module_to_your_html) type to get some more [sane defaults](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#other_differences_between_modules_and_standard_scripts). Can we also format this so the JS is on multiple lines and indented?